task/06-conditions-n-loops-tasks.js

```diff
@@ -251,7 +309,23 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    const stringCCN = [...ccn.toString()].reverse();
+
+    const modifiedCCN = stringCCN.map((value, index) => {
+        if (index % 2) {                     //  double every second number
+            if (value * 2 > 9) {                 //  and if it is more than 9- subtract 9
```

Comment:
You can use inversion of `if` to simplify code (all `if` must be at the same level).